src/package/nuspec/Microsoft.TestPlatform.nuspec

```diff
@@ -388,5 +390,38 @@
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Extension.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Extension.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Extension.MSAA.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Extension.MSAA.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.ExtensionUtilities.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.ExtensionUtilities.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Framework.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Framework.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Logging.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Logging.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Playback.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Playback.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.WindowsStoreUtility.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.WindowsStoreUtility.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITesting.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITesting.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.Interop.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.Interop.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Playback.Engine.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Playback.Engine.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Synchronization.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Synchronization.dll" />
+    <file src="net451\$Runtime$\UIAComWrapper.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\UIAComWrapper.dll" />
+
+    <file src="net451\$Runtime$\IEPropertyConfiguration.xml"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\CUITPlugins\IEPropertyConfiguration.xml" />
```

Comment:
Should it not be src="net451\$Runtime$\CUITPLugins\IEPropertyConfiguration.xml"

Reply:
good catch. it failed locally fixing it